### PR TITLE
Add OpenAPI shortcodes documentation

### PR DESCRIPTION
**Pull request by grafsean**
Add documentation for the `docs/openapi/info` and `docs/openapi/path` shortcodes.

### CODE REVIEW

File: docs/sources/write/shortcodes/index.md [hunk 1]
```diff
@@ -412,6 +412,71 @@ This isn't rendered.
...
+```markdown
+{{/*% docs/openapi/info data="<SPECIFICATION_FILENAME>" %*/}}
+```
+
+### Examples
+
+Display the API information for a remote specification at `https://raw.githubusercontent.com/OAI/OpenAPI-Specification/main/examples/v3.0/petstore-expanded.json`:
+
+```markdown
+{{/*% docs/openapi/info url="https://grafana.com/OAI/OpenAPI-Specification/main/examples/v3.0/petstore-expanded.json" %*/}}
```
Comment (jdbaldry) [inline docs/sources/write/shortcodes/index.md:441]: This URL doesn't match the lead-in text.

**Review by jdbaldry — COMMENTED** [review]
Should the snippets not use `{{<` and `>}}` instead of `{{%` and `%}}`?

The comments need to come after the shortcode delimiters:

```markdown
{{%/* docs/openapi/info data="<SPECIFICATION_FILENAME>" */%}}
```

instead of:

```
{{/*% docs/openapi/info data="<SPECIFICATION_FILENAME>" %*/}}
```

File: docs/sources/write/shortcodes/index.md [hunk 2]
```diff
@@ -412,6 +412,71 @@ This isn't rendered.
...
+
+| Parameter | Description                                                                                       | Required |
+| --------- | ------------------------------------------------------------------------------------------------- | -------- |
+| `url`     | The URL of the OpenAPI specification to fetch.                                                    | no       |
+| `data`    | The filename of the OpenAPI specification to use from the website `data/docs/openapi/` directory. | no       |
+
+To fetch a remote specification from a URL:
+
+```markdown
+{{/*% docs/openapi/info url="<SPECIFICATION_URL>" %*/}}
```
Comment (jdbaldry) [inline docs/sources/write/shortcodes/index.md:427]: https://github.com/grafana/writers-toolkit/pull/833#pullrequestreview-2279793232

For example,
```suggestion
{{</* docs/openapi/info url="<SPECIFICATION_URL>" */>}}
```

The same applies for all the snippets

**Review by jdbaldry — APPROVED** [review]
LGTM, thanks for the docs!

**Comment by grafsean** [comment]
Thank you for the review Jack!

**Pull request merged**